src/Main.java

```diff
@@ -19,7 +27,11 @@ public static void main(String[] args) {
                     System.out.println("password");
                     String inputPassword = scanner.nextLine();
                     LoginCommand loginCommand = new LoginCommand();
-                    loginCommand.execute(inputUsername, inputPassword);
+                    try {
+                        loginCommand.execute(inputUsername, inputPassword);
+                    } catch (LoginCommandException e) {
+                        throw new RuntimeException(e);
+                    }
```

Comment:
there will most likely be an error here)

Reply:
remove `try-catch` here